client/src/components/questions/QandA.jsx

```diff
@@ -20,23 +24,35 @@ const QandA = () => {
       headers: {Authorization: API_KEY}
     })
     .then((response) => {
-      console.log(response.data.results);
+      //console.log(response.data.results);
       setQuestions(response.data.results);
+      setMasterList(response.data.results);
     })
     .catch((err) => {
       //console.log(err);
     })
-
   }, [])
 
```

Comment:
I think you need semicolons after lines 25, 30, and 34

Reply:
good catch I will fix those